services/web-app/pages/assets/[host]/[org]/[repo]/[library]/[ref]/[asset].js

```diff
@@ -175,56 +186,40 @@ const Asset = ({ libraryData }) => {
                 </Grid>
               </DashboardItem>
             </Column>
-            <Column className={dashboardStyles.column} sm={0} md={1}>
+            <Column className={dashboardStyles.column} sm={0} md={4}>
               <DashboardItem
                 aspectRatio={{ md: '2x1', lg: '16x9', xlg: '2x1' }}
                 border={['sm', 'md', 'lg', 'xlg']}
+                href={`${githubRepoUrl}/issues/?q=is%3Aissue+is%3Aopen+in%3Atitle+${assetData.content.name}`}
               >
                 <dl>
-                  <dt className={dashboardStyles.label}>Coming soon...</dt>
-                  <dd className={dashboardStyles.labelLarge}>–</dd>
-                </dl>
-              </DashboardItem>
-            </Column>
-            <Column className={dashboardStyles.column} sm={0} md={1}>
-              <DashboardItem
-                aspectRatio={{ md: '2x1', lg: '16x9', xlg: '2x1' }}
-                border={['sm', 'md', 'lg', 'xlg']}
-                href={libraryPath}
-              >
-                <dl>
-                  <dt className={dashboardStyles.label}>Coming soon...</dt>
-                  <dd className={dashboardStyles.labelLarge}>–</dd>
+                  <dt className={dashboardStyles.label}>Open issues</dt>
+                  <dd className={dashboardStyles.labelLarge}>
+                    {assetData.content.issueCount || 0}
```

Comment:
```suggestion
                    {assetData.content.issueCount || '-'}
```

Reply:
We decided on the webex that 0 made more sense than just -, so a user can still click the link and see the issue count in github is actually 0.

Reply:
ohhh ok missed that 👍